jetty-ee10/jetty-ee10-demos/jetty-ee10-demo-embedded/src/main/resources/fileserver.xml

```diff
@@ -25,10 +25,14 @@
                   <Array type="String"><Item>index.html</Item></Array>
                 </Set>
                 <Set name="baseResource">
```

Comment:
interesting, this is still `baseResource`, so we've not renamed it everywhere... or this XML has not been tested.